scripts/mgear/maya/shifter/component/guide.py

```diff
@@ -263,7 +263,9 @@ def setFromHierarchy(self, root):
             else:
                 node = dag.findChild(self.model, self.getName(name))
                 if not node:
-                    mgear.log("Object missing : %s"%name, mgear.sev_warning)
+                    mgear.log("Object missing : %s" % (
+                        self.getName(name), mgear.sev_warning)
```

Comment:
I spotted a problem here!

```python
mgear.log("Object missing : %s" % (
   self.getName(name), mgear.sev_warning)
)
```

Should have been..

```python
mgear.log("Object missing : %s" % (
    self.getName(name)), mgear.sev_warning
)
```

Note the parenthesis difference.

Without it, you'll see this error:

```bash
# TypeError: not all arguments converted during string formatting #
```

Reply:
Oh! I see, thanks!  I will cherrypick this change :)

Reply:
Just found that the revision on a commit is already apply. It is the first time that I have received a revision from a pull request. I didn't know this is automatic. :P

Reply:
oh! I don't know in what I was thinking yesterday, now I see the error. I will fix it in a new commit :(